src/App.js

```diff
@@ -16,19 +16,32 @@ function App() {
   // Your functions should accept a parameter of the the item data being displayed to the DOM (ie - should recieve 5 if the user clicks on
   // the "5" button, or the operator if they click one of those buttons) and then call your setter function to update state.
   // Don't forget to pass the functions (and any additional data needed) to the components as props
+const [disNum, newDisNum] = useState('')
+
+const addNum = (num) => {
+    newDisNum(disNum => disNum + num)
```

Comment:
I'm seeing this pattern where people are passing in a callback function/arrow function.  Where are we seeing this pattern?  What is the purpose of the callback?